### PR TITLE
Hide most Buttons

### PR DESCRIPTION
**Pull request by marfranke**

### CODE REVIEW

**Comment by Ablu** [comment]
Uh nice! I think we should find a way to force it into tileview before merging it though... Otherwise one cannot switch modes anymore. Currently, because jitsi seems to remember the last state it will toggle between each page load :/ (that was my attempt to force it into tile view).

File: lib/gemeinsam_sport_frontend_web/templates/room/room.html.eex [hunk 1]
```diff
@@ -57,6 +57,38 @@ document.body.onload = function() {
...
+            TOOLBAR_BUTTONS: [
+                'microphone',
+                'camera',
+                //'closedcaptions',
+                //'desktop',
+                'fullscreen',
+                //'fodeviceselection',
+                'hangup',
+                //'profile',
+                //'info',
```
Comment (Ablu) [inline lib/gemeinsam_sport_frontend_web/templates/room/room.html.eex]: I think you can leave out the disabled ones... If we keep them for reference we can never be sure whether the list is still up-to-date. But I do not mind too much...

**Comment by Ablu** [comment]
In general I would prefer that we would use forks for the merge requests and only use branches on the main repo for longer running feature branches. That way we do not clutter the history too much...

Also it accidently triggered a push to the production page :P. But that is mostly my fault in configuring the actions wrongly...

File: lib/gemeinsam_sport_frontend_web/templates/room/room.html.eex (hunk 1 above)
Comment (marfranke) [reply lib/gemeinsam_sport_frontend_web/templates/room/room.html.eex]: Replaced it with a comment to the source

**Comment by marfranke** [comment]
> Uh nice! I think we should find a way to force it into tileview before merging it though... Otherwise one cannot switch modes anymore. Currently, because jitsi seems to remember the last state it will toggle between each page load :/ (that was my attempt to force it into tile view).

Good point. Used the event to check that.

**Comment by marfranke** [comment]
> In general I would prefer that we would use forks for the merge requests and only use branches on the main repo for longer running feature branches. That way we do not clutter the history too much...
> 
> Also it accidently triggered a push to the production page :P. But that is mostly my fault in configuring the actions wrongly...

Ok. Will do that next time

**Comment by Ablu** [comment]
LGTM. Do you want to squash c8f5ccea4110ccb5cef738af52ba6fa1e5a16a07 and b8f94d50bb48e77eff9c9c000bf130a7bc3dedcb?

**Pull request merged**